docs/operatorTestPlaybooksCI.md

```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
```

Comment:
Try this: `This document describes the Upstream Operator Test Playbooks CI Pipeline and how it works`